src/importRemote/src/index.ts

```diff
@@ -52,18 +51,39 @@ export const importRemote = async <T>({
       await loadRemote(`${url}/${remoteEntryFileName}`, scope, bustRemoteEntryCache);
       if (!window[scope]) {
         throw new Error(
-          `Remote loaded succesfully but ${scope} could not be found! Verify that the name is correct in the Webpack configuration!`,
+          `Remote loaded successfully but ${scope} could not be found! Verify that the name is correct in the Webpack configuration!`,
         );
       }
       // Initializes the share scope. This fills it with known provided modules from this build and all remotes
-      await __webpack_init_sharing__("default");
-      // Initialize the container, it may provide shared modules:
-      await window[scope].init(__webpack_share_scopes__.default);
+      if(!__webpack_share_scopes__?.default) {
```

Comment:
Are you sure about this check? I debugged it and `__webpack_share_scopes__.default` is always populated so `await __webpack_init_sharing__("default")` never gets called. Strange thing is everything works without it too...